recipes/aquila_umap/build.sh

```diff
@@ -0,0 +1,2 @@
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Please move this to the `script:` line under the `build:` section in meta.yaml. It will look like:

    build:
        noarch: python
        number: 0
        script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"

Reply:
thank you!